resources/styles/components/task-plan/homework/exercises.less

```diff
@@ -54,6 +54,16 @@ table.exercise-table {
     * + * {
       display: none;
     }
+
+    img {
+      display: block;
+      width: 100%;
```

Comment:
I don't think `display: block` is strictly needed here.  I tested locally and images still displayed scaled in the table with the rule disabled at least.  Maybe there's other rules you're working around that I don't see though